Userland/Libraries/LibJS/Runtime/Object.h

```diff
@@ -211,6 +214,7 @@ class Object : public Cell {
     Shape* m_shape { nullptr };
     Vector<Value> m_storage;
     IndexedProperties m_indexed_properties;
+    HashMap<StringOrSymbol, IntrinsicAccessor> m_intrinsics;
```

Comment:
Instead of growing every `JS::Object` by `sizeof(HashMap)` for something that's only used by global object, can we find another way to store this?

For example, a global `HashMap<JS::Object*, NonnullOwnPtr<HashMap<StringOrSymbol, IntrinsicAccessor>>` keyed by the global object pointer. There might be better strategies too :)